lib/semantics/type.cc

```diff
@@ -270,7 +270,7 @@ void ExprResolver::Resolve(Symbol &symbol) {
   if (auto *type{symbol.GetType()}) {
     if (type->category() == DeclTypeSpec::TypeDerived) {
       DerivedTypeSpec &dts{type->derivedTypeSpec()};
-      for (auto & [ name, value ] : dts.paramValues()) {
+      for (auto &[name, value] : dts.paramValues()) {
```

Comment:
I formatted this originally with clang-format version 5.0.1. I've switched to 6.0.1 and now it matches yours.